plugins/newrelic-dashboard/src/Router.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Entity } from '@backstage/catalog-model';
+import React from 'react';
+import { MissingAnnotationEmptyState } from '@backstage/core-components';
+import { Button } from '@material-ui/core';
+import { NewRelicDashboard } from './components/NewRelicDashboard';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { NEWRELIC_GUID } from './constants';
+
+export const isNewRelicDashboardAvailable = (entity: Entity) =>
+  Boolean(entity?.metadata?.annotations?.[NEWRELIC_GUID]);
+
+export const Router = () => {
+  const { entity } = useEntity();
+
+  if (isNewRelicDashboardAvailable(entity)) {
+    return <NewRelicDashboard />;
+  }
+
+  return (
+    <>
+      <MissingAnnotationEmptyState annotation={NEWRELIC_GUID} />
+      <Button variant="contained" color="primary" href="#">
```

Comment:
should this link to somewhere?

Reply:
Once this is merged, We can link it to the readme file.

Reply:
Could make it the proper link already now though (just take the link of any other plugin and edit it)